src/kOS/Binding/FlightControlManager.cs

```diff
@@ -225,6 +227,15 @@ public void SelectAutopilotMode(VesselAutopilot.AutopilotMode autopilotMode)
             }
         }
 
+        public string GetAutopilotModeName()
+        {
+            // TODO: As of KSP 1.0.4, RadialIn and RadialOut are swapped.  Check if still true in future versions.
+            if (currentVessel.Autopilot.Mode == VesselAutopilot.AutopilotMode.RadialOut) { return "RadialIn"; }
+            if (currentVessel.Autopilot.Mode == VesselAutopilot.AutopilotMode.RadialIn) { return "RadialOut"; }
+
+            return currentVessel.Autopilot.Mode.ToString();
```

Comment:
Of course, if it's better for the way these are represented internally (I don't have too much idea about this), since these functions are only used for these bindings.